src/content/guides/compatible-terminals.mdx

```diff
@@ -0,0 +1,149 @@
+---
+title: Compatible Terminals
+description: Compatible payment terminals.
+nav:
+  path: Reference/Merchant Integrations
+  order: 1
+---
+import Disclosure from '../../components/Disclosure.vue';
+import Faq from '../../components/Faq.astro';
+
+The Centrapay service is supported on a range payment and point of sale devices listed below. If the device model is not in this list, it is not supported. If you would like to discuss a new integration or support of Centrapay on a new device, please contact Centrapay at support@centrapay.com
+
+## Terminal Providers
+
+ <Faq>
+  <div>
+    <Disclosure client:visible title="Verifone">
+      Android T650C
```

Comment:
@Wulingoh You can't search "Android T650C" on the site and find this page. What would be involved to make this happen?

Reply:
@tom-blake The search function is limited to headings only. The dropdown list titles are not headings; they reside within the dropdown itself. Creating headings for these would disrupt the component's structure.